src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -22,7 +22,7 @@
         "symfony/dependency-injection": "~2.8|~3.0",
         "symfony/config": "~2.8|~3.0",
         "symfony/event-dispatcher": "~2.8|~3.0",
-        "symfony/http-foundation": "~2.8|~3.0",
+        "symfony/http-foundation": "~3.1",
```

Comment:
@xabbuh is this the best way to do it? I see at least one other component has a `~3.0` requirement, I assume this is expected to change as new functionality is added to Symfony 3